co2eq/electricity/emissionfactors.json

```diff
@@ -0,0 +1,122 @@
+{
+  "factors": [
+    { "country": "CAN", "state": "QC", "factor": 1.5 },
```

Comment:
do you have access to the year of the data source? Might be relevant to put it in the json

Reply:
added